driver-core/src/test/java/com/datastax/driver/core/CCMBridge.java

```diff
@@ -190,6 +193,7 @@
     } else if (inputScyllaVersion != null && !inputScyllaVersion.trim().isEmpty()) {
       installArgs.add(" --scylla ");
       installArgs.add("-v release:" + inputScyllaVersion);
+      envMap.put("SCYLLA_EXT_OPTS", "--smp 2");
```

Comment:
this is going to be problematic for github action, since some test spin up 3 nodes

Reply:
I don't think that this is a problem, as GitHub Actions workers have 2 cores. The test works correctly, apart from unrelated tests which fail due to this change (described in my next comment).